Simperium/build.gradle

```diff
@@ -1,23 +1,9 @@
-apply plugin: 'com.android.library'
-apply plugin: 'com.novoda.bintray-release' // must be applied after your artifact generating plugin (eg. java / com.android.library)
-apply plugin: 'maven'
-
-buildscript {
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.2'
-        classpath 'com.novoda:bintray-release:0.9.1'
-    }
-
-    repositories {
-        google()
-        jcenter()
-    }
+plugins {
```

Comment:
I've updated this project to Gradle 7.1.1, Android Gradle Plugin 4.2.2 and to use plugin DSL. We'd have wanted to do this anyway, so I figured I'd included it in this update.

Reply:
Great. You save me headaches for sure. Thanks!